src/relayclient/ContractInteractor.ts

```diff
@@ -137,65 +151,52 @@ export default class ContractInteractor {
   }
 
   async getForwarder (recipientAddress: Address): Promise<Address> {
-    const recipient = await this._createRecipient(recipientAddress)
+    const recipient = await this._createKnowsForwarder(recipientAddress)
```

Comment:
this function will crash if the client doesn't support the interface.
if you require clients to opt-in for interface to support it, at least give a proper error incase it doesn't.